packages/modules/order/src/migrations/Migration20240219102530.ts

```diff
@@ -115,37 +115,37 @@ export class Migration20240219102530 extends Migration {
       CREATE INDEX IF NOT EXISTS "IDX_order_display_id" ON "order" (
```

Comment:
Q: when handling migration changes, do you use the command or do you do it manually? Cause if the snapshot are not up to date we will never be able to use the commands again :D so many at some point we need to be sure that all modules snapshots are up to data wdyt?

Reply:
I always do it manually.
yes, I think we have to have just 1 single initial migration file per module with the final snapshot updated before releasing them in v2.